src/app/pages/SearchResultsPage/SearchResultsList.tsx

```diff
@@ -42,18 +40,17 @@ export const SearchResultsList: FC<{
   networkForTheme: Network
   searchResults: SearchResults
   tokenPrices: AllTokenPrices
-}> = ({ title, networkForTheme, searchResults, tokenPrices }) => {
+}> = ({ title, searchResults, tokenPrices }) => {
   const { t } = useTranslation()
 
   const numberOfResults = searchResults.length
 
   if (!numberOfResults) {
     return null
   }
-  const theme = getThemeForScope(networkForTheme)
 
   return (
-    <ResultListFrame theme={theme}>
```

Comment:
we need to keep this component to distinguish search results between networks. Figma designs were created for mainnet only I think